rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -765,6 +765,13 @@ error_code sys_fs_opendir(ppu_thread& ppu, vm::cptr<char> path, vm::ptr<u32> fd)
 		// Add real directories
 		while (dir.read(data.emplace_back()))
 		{
+			if (!data.back().is_directory && data.back().name.starts_with(reinterpret_cast<const char*>(u8"＄")))
```

Comment:
I think it's already handled in vfs::unescape

Reply:
I see, changed the check.